### PR TITLE
CSES Christmas Party Sol

### PR DESCRIPTION
**Pull request by HarryWangATX**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/cses-1717.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.util.*;
+import java.io.*;
+
+public class ChristmasParty {
```
Comment (dongliu0426) [inline solutions/cses-1717.mdx:29]: use tabs

**Review by dongliu0426 — COMMENTED** [review]
good but this problem is already mentioned in the [Gold combo module](https://usaco.guide/gold/combo?lang=cpp#method-2-dynamic-programming). maybe just link [this](https://usaco.guide/gold/combo?lang=cpp#derangements)? @SansPapyrus683 opinions?

**Comment by bqi343** [comment]
Isn't this problem a duplicate of https://usaco.guide/gold/combo/?lang=cpp#derangements ?

**Comment by bqi343** [comment]
> good but this problem is already mentioned in the [Gold combo module](https://usaco.guide/gold/combo?lang=cpp#method-2-dynamic-programming). maybe just link [this](https://usaco.guide/gold/combo?lang=cpp#derangements)?

I think we should remove this CSES problem from the module.

**Comment by HarryWangATX** [comment]
Yeah, it's pretty redundant. I'll update this to remove the problem.

**Review by bqi343 — APPROVED** [review]

**Pull request merged**